app/models/factories/ivl_eligibility_factory.rb

```diff
@@ -222,8 +222,9 @@ def prioritized_csr(csr_kinds)
 
     def fetch_csr
       return {:csr => 'csr_0'} if (shopping_tax_members.count != shopping_member_ids.count) || any_aptc_ineligible?
+      # picks csr_kind based on individual level
 
-      csr_kinds = tax_households.map(&:current_csr_eligibility_kind)
+      csr_kinds = tax_households.last.eligibile_csr_kind(shopping_member_ids)
       {:csr => prioritized_csr(csr_kinds)}
```

Comment:
Do we need this since we are using `eligibile_csr_kind` which would give only one csr_kind based on the new rule already?